py/picca/pk1d/postproc_pk1d.py

```diff
@@ -221,6 +227,24 @@ def compute_mean_pk1d(
     if "sub_forest_id" in p1d_table_cols:
         p1d_table_cols.remove("sub_forest_id")
 
+    if "Delta_k_real" in p1d_table_cols:
```

Comment:
Why are those cols removed (esp given that you just explicitely added them to the readin part above)? is it because some iterations further down go over all existing cols? In which case maybe a check there would be more compact (and more clearly tell why this is done) as in `if "Delta_" in col: continue`

Reply:
The explicit part above is just for renaming. The Table reads all the file.
And yes there is an iteration further down that go over all columns. We do not want to average Detla_k for nothing.
This is quite equivalent no ? I can change that though

Reply:
After checking more carefully the code, your change would modify a lot of things, since the statistics array are first create in a list and then computed in another. 
I will simplify the way I wrote it.

Reply:
I was mostly confused because these fields have not been used anywhere so-far, yet they are first renamed further up (which I now noticed works from uppercase as probably stored in the FITS to the actual case used for those variables) and then removed...
I guess explicitely writing this down makes it clearer. Else a comment in the file such as "#remove columns we don't need averages of" might be helpful to clarify...